verifier/src/report_body.rs

```diff
@@ -114,13 +114,57 @@ impl<T: Accessor<ConfigId>> Verifier<T> for ConfigIdVerifier {
     }
 }
 
+impl Accessor<ConfigSvn> for ReportBody {
+    fn get(&self) -> ConfigSvn {
+        self.config_svn()
+    }
+}
+
+impl Accessor<ConfigSvn> for ConfigSvn {
+    fn get(&self) -> ConfigSvn {
+        self.clone()
+    }
+}
+
+/// Verify the [`ConfigSvn`] is as expected.
+#[derive(Clone, Debug, Eq, Hash, PartialEq)]
+pub struct ConfigSvnVerifier {
+    expected_svn: ConfigSvn,
+}
+
+impl ConfigSvnVerifier {
+    /// Create a new [`ConfigSvnVerifier`] instance.
+    ///
+    /// # Arguments:
+    /// * `expected_svn` - The expected svn.
+    pub fn new(expected_svn: ConfigSvn) -> Self {
+        Self { expected_svn }
+    }
+}
+
+impl<T: Accessor<ConfigSvn>> Verifier<T> for ConfigSvnVerifier {
+    type Error = VerificationError;
+
+    fn verify(&self, evidence: &T) -> CtOption<Self::Error> {
+        let expected = self.expected_svn.clone();
+        let actual = evidence.get();
+
+        let is_some = actual.as_ref().ct_lt(expected.as_ref());
```

Comment:
Does this need the `as_ref()`s?

Reply:
Yes, because ConfigSvn doesn't implement ConstantTimeEq and ConstantTimeGreater, so the `as_ref()` gets to the underlying u16

Reply:
added more info to https://github.com/mobilecoinfoundation/sgx/issues/280